diagnostic_common_diagnostics/CMakeLists.txt

```
@@ -21,6 +22,10 @@ if(BUILD_TESTING)
         test_ntp_monitor
         test/systemtest/test_ntp_monitor.py
         TIMEOUT 10)
+    ament_add_pytest_test(
+        test_cpu_monitor
+        test/systemtest/test_cpu_monitor.py
```

```suggestion
        test_cpu_monitor
        test/systemtest/test_cpu_monitor.py
        TIMEOUT 10)
    ament_add_pytest_test(
        test_ntp_monitor
        test/systemtest/test_ntp_monitor.py
```

Alphabetical ordering